<launch>
    <arg name="egm_settings" default="" doc="The filename (including path) of the file containing the EGM settings to apply for each EGM session." />
    <arg name="taskname" default="T_ROB1" doc="The name of the RAPID task which is running the 'StateMachine Add-In'" />
    <arg name="rws_namespace" default="/rws" doc="Namespace of the RWS services and topics to be used by the state machine" />
    <arg name="egm_namespace" default="/egm" doc="Namespace of the EGM services and topics to be used by the state machine" />
    
    <param name="egm_settings" value="$(arg egm_settings)"/>
    <param name="taskname" value="$(arg taskname)"/>
    <param name="rws_namespace" value="$(arg rws_namespace)"/>
    <param name="egm_namespace" value="$(arg egm_namespace)"/>
    <node name="abb_robot_driver_state_machine" pkg="abb_robot_driver_state_machine" type="state_machine_node"/>
</launch>
